src/styles/main/_year-control.scss

```diff
@@ -65,3 +65,63 @@
     }
   }
 }
+
+.year-dropdown {
+  display: none;
+}
+
+@media only screen and (max-width: 850px) {
```

Comment:
Where is the `max-width: 850px` coming from? Can we use the regular set of breakpoints in `_grid-settings.scss` and organize it as default CSS small, then supplement when larger? This looks like the opposite.